src/main/java/org/candlepin/subscriptions/conduit/inventory/kafka/KafkaEnabledInventoryService.java

```diff
@@ -97,6 +102,14 @@ protected void sendHostUpdate(List<ConduitFacts> facts) {
         }
     }
 
+    @Recover
+    protected void recover(Throwable throwable) {
+        log.info("sendHostUpdate recover");
+        log.info(String.format("Exception type: %s", throwable.getClass().getName()));
```

Comment:
https://sonarcloud.io/project/issues?branch=ent-1573&id=rhsm-subscriptions&open=AXeS8lvgQy8CWpD6Pw_Y&resolved=false&types=CODE_SMELL

This syntax should make it stop complaining:

```java
log.info("Exception type: {}", throwable.getClass().getName());
log.info("Exception message: {}", throwable.getMessage()));
```

Reply:
I added a conditional instead, I hope that works.

Reply:
February 11, 2021, 4:42 PM
https://sonarcloud.io/project/issues?id=rhsm-subscriptions&pullRequest=289&resolved=false&sinceLeakPeriod=true

Reply:
Yep, Queuing up next commit now...

Reply:
@dmckoske please don't do that. Please use the syntax that @lindseyburnett pointed out instead when logging.